### PR TITLE
fix(datastore): Adding tracing to run method

### PR DESCRIPTION
**Pull request by bhshkh**
Find info in https://github.com/googleapis/google-cloud-go/issues/9464

### CODE REVIEW

**Comment by notcool11** [comment]
thank you!

do you know if i will get notified for if/when this PR makes it into a release so i can pull that version of the package? thanks again!

**Comment by bhshkh** [comment]
> thank you!
> 
> do you know if i will get notified for if/when this PR makes it into a release so i can pull that version of the package? thanks again!

Once the PR is merged, it will be included in the next release which is 1.16.0

You can subscribe to https://github.com/googleapis/google-cloud-go/pull/8711 to get notified.

**Review by gkevinzheng — APPROVED** [review]

**Comment by bhshkh** [comment]
Do not merge until release freeze ends in mid April

**Pull request merged**
